recipes/dmd/meta.yaml

```diff
@@ -0,0 +1,23 @@
+package:
+  name: dmd
+  version: "2.065.0"
+
+source:
+  fn: dmd.2.065.0.zip
+  url: http://downloads.dlang.org/releases/2014/dmd.2.065.0.zip
```

Comment:
I believe so, but I never did it and I do not have the time right now to do so. I am adding this specific version of the D so I can satisfy a dependency for another tool. 

I could not find a D compiler in conda.